src/types/tuple.rs

```diff
@@ -25,6 +25,16 @@ pub fn types(schema: &Schema) -> Vec<LogicalType> {
         .map(|column| column.datatype().clone())
         .collect_vec()
 }
+pub fn full_columns(schema: &Schema) -> Vec<(usize, ColumnRef)> {
```

Comment:
`full_columns` & `column_names` It seems that these two methods are not necessary as methods. It is better to remove them.